src/codeview.js

```diff
@@ -556,10 +572,35 @@ var Codeview = GObject.registerClass({
         }
     }
 
+    dbusRegister(appId, id) {
+        const objPath = Gio.Application.get_default().get_dbus_object_path();
+        const winName = appId.replace(/\./gi, '_');
```

Comment:
I was thinking that we should use the window id here just to follow the same object path pattern used in the ToolboxWindow, but I am not sure how you could know the toolbox window id from the Clubhouse. If you find the way, better to do that, IMO.